### DISCUSSION TITLE
how to change the thickness of produced boundary boxes.

### DISCUSSION THREAD

**Discussion by alaa-shubbak** [open-mmlab/mmdetection#9314] [Q&A] [answered]
### What's the feature?

Hello , 

Thanks for your effort . after training my model with my custom dataset, i use the function `python tools/test.py`   to produce a results of images that contain the boundary boxes as bellow. 

![images ](https://user-images.githubusercontent.com/10245810/198042230-3e8c9321-cc04-4d2e-b24f-e995a5add4e8.jpg)

some images look good , while another look not so clear. 
I would like to ask about the possibility to have such bbox more thick and with more obvious color. 

I try to find a way, I read the script `tools/test.py` but i  could not figure it out. I don't know where shall i search or update them or change them. 

any help will be highly apricated.

**Comment by wanghonglie** [comment, marked as answer]
Which version of mmdet do you use? If the version is 2. x, you can try to modify the parameters(`bbox_color`and`thickness`) of the method `show_results`:

https://github.com/open-mmlab/mmdetection/blob/e71b499608e9c3ccd4211e7c815fa20eeedf18a2/mmdet/apis/inference.py#L248

**Reply by alaa-shubbak** [reply]
this is the results after running `collect_env.py`

<img width="770" alt="image" src="https://user-images.githubusercontent.com/10245810/203860082-e2ef3900-7479-4861-9906-032721650216.png">

I think i am using  mmdet 2.25 

i changed the `text_color =(0,0,0)` and `text_color =(255,255,255)` , but it does not work. 

for example , i want the text to be white , what shall i write? although the thickness option is not there.  
what shall i do ?
